mozilla-release/mobile/android/locales/en-US/chrome/aboutHome.dtd

```diff
@@ -4,4 +4,6 @@
 
 <!-- This string should be kept in sync with the home_title string
      in android_strings.dtd -->
-<!ENTITY abouthome.title                  "&brandShortName; Home">
+<!-- Cliqz start -->
+<!ENTITY abouthome.title                  "Start Tab">
```

Comment:
@spacifici
I think you can add a string in cliqz_strings.dtd and refer to it here, so we can keep all translation strings into one file.